microcloud/test/includes/microcloud.sh

```diff
@@ -74,6 +77,23 @@ $(true)                                                 # workaround for set -e
 "
 fi
 
+if [ -z "${IGNORE_CEPH_NETWORKING}" ]; then
+  if [ -n "${CEPH_PUBLIC_NETWORK}" ]; then
+    setup="${setup}
+${CEPH_PUBLIC_NETWORK}"                                 # setup public network
+  else
+    setup="${setup}
+_ENTER_"
```

Comment:
I'm guessing the `_ENTER_` is to just select the default value that MicroCloud will find? I think that might make it hard to validate what the outcome is.

I think for the test suite, we should always explicitly enter values so that we can pass the expected value to the `validate_microceph` function. 

One thing that might be useful in a future PR is an additional test of MicroCloud's default values in general, probably by parsing the `out` file which contains the stdout from `microcloud init`.